app-admin/terraform/metadata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>amynka@gentoo.org</email>
+		<name>Amy Liffey</name>
+	</maintainer>
+	<maintainer type="person">
+		<email>dan@danmolik.com</email>
+		<name>Dan Molik</name>
+	</maintainer>
+	<longdescription lang="en">
```

Comment:
this is way too much long longdescription

Reply:
What length and content were you thinking?

Reply:
I cut it down to the first 2 paragraphs